src/main/java/duke/command/AddTodoCommand.java

```diff
@@ -0,0 +1,37 @@
+package duke.command;
+
+import duke.exception.MissingArgumentException;
+import duke.storage.TaskList;
+import duke.task.Todo;
+
+/**
+ * Class use to handle command: add todo.
+ * Allows user to add todo task into the task list.
+ */
+public class AddTodoCommand extends Command {
+    private String request;
+
+    /**
+     * Constructor to create add todo task according to user's request
+     * @param request user's request to be processed into todo task.
+     */
+    public AddTodoCommand(String request) {
+        this.request = request;
+    }
+
+    @Override
+    public String execute(TaskList tasks) throws MissingArgumentException {
+
+        String[] task = request.strip().split("todo");
+
+        // check missing description of todo task
+        if (task.length == 0) {
+            throw new MissingArgumentException("☹ OOPS!!! The description of a todo cannot be empty.");
```

Comment:
Instead of having this message as parameter in your constructor, if this is the default string for this exception it might be better to put it inside the exception itself